### PR TITLE
WRO-442: Add or fix touch support for sandstone components

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Add or fix touch support for sandstone components

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
`sandstone/Alert` and `sandstone/Input`: Add support for vertical screen
`sandstone/Icon`: Add a new icon wallpaper
`sandstone/VideoPlayer`: Add prop to exit player by touching back button
`sandstone/RadioItem`: Replace onTap with onClick to get focus and recieve event function
`sandstone/Picker` and `sandstone/Date(Time)Picker`: Fix picker to support touch input
`sandstone/Scroller` and `sandstone/VirtualList`: Fix default overscrollEffect(drag) for scroller and virtualList

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-442

### CODE REVIEW

File: internal/DateComponentPicker/DateComponentPicker.js [hunk 1]
```diff
@@ -128,7 +128,6 @@ const DateComponentPickerBase = kind({
 			css={css}
 			data-webos-voice-labels-ext={voiceLabel}
 			index={value}
-			joined
```
Comment (0x64) [inline internal/DateComponentPicker/DateComponentPicker.js:131]: Let's double-check with the UX team whether this is the right way or not.

File: internal/DateComponentPicker/DateComponentRangePicker.js [hunk 2]
```diff
@@ -108,7 +108,6 @@ const DateComponentRangePickerBase = kind({
 			accessibilityHint={(accessibilityHint == null) ? label : accessibilityHint}
 			css={css}
 			data-webos-voice-labels-ext={voiceLabel}
-			joined
```
Comment (0x64) [inline internal/DateComponentPicker/DateComponentRangePicker.js:111]: Let's double-check with the UX team whether this is the right way or not.

**Review by 0x64 — APPROVED** [review]
Looks good to me.
Needs to check UX before merging.

File: samples/sampler/stories/default/VideoPlayer.js [hunk 3]
```diff
@@ -135,6 +136,7 @@ export const _VideoPlayer = () => {
 			</label>
 			<VideoPlayer
 				autoCloseTimeout={number('autoCloseTimeout', Config, 7000)}
+				backButtonAriaLabel={text('backButtonAriaLabel', Config)}
```
Comment (jeonghee27) [inline samples/sampler/stories/default/VideoPlayer.js:139]: I think you missed above
`text('backButtonAriaLabel', _VideoPlayer, Config, 'go to previous');`

https://github.com/enactjs/sandstone/pull/1145/files#diff-17238cddc27f30bf52c486b94901be1f84908c67c7b534aebd9b4e9d952484cc

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,24 @@
...
+
+### Added
+
+- `sandstone/Alert` and `sandstone/Input` support for portrait mode
+- `sandstone/Icon` supported icon list, adding a new icon `wallpaper`
+- `sandstone/VideoPlayer` props `backButtonAriaLabel` and `onBack` to provide a way to exit video player via touch
+
+### Changed
+
+- `sandstone/Panels.Header` and `sandstone/RadioItem` to use `onClick` instead of `onTap` for touch support
```
Comment (jeonghee27) [inline CHANGELOG.md:15]: Why do you exclude Sample fix?
https://github.com/enactjs/sandstone/pull/793/files#diff-e8f6b68eaefaf44020bc3ff1fc7a18f1a73690ff91da72e19756be6bf9edef04

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,24 @@
...
+- `sandstone/DatePicker` and `sandstone/TimePicker` to not show press effect via touch input
+- `sandstone/Scroller` and `sandstone/VirtualList` to show overscroll effect when flicking
```
Comment (jeonghee27) [inline CHANGELOG.md:17]: Don't we need the modification of useScroll/useOverscrollEffect.js?
https://github.com/enactjs/sandstone/pull/792/files

CC: @0x64

Comment (0x64) [reply CHANGELOG.md:17]: @jeonghee27 we already have the logic change in [release/2.0.x branch](https://github.com/enactjs/sandstone/blob/release/2.0.x.develop/useScroll/useOverscrollEffect.js#L32).

File: CHANGELOG.md (hunk 4 above)
Comment (juwonjeong) [reply CHANGELOG.md:15]: Samples fix are added.

File: samples/sampler/stories/default/VideoPlayer.js (hunk 3 above)
Comment (juwonjeong) [reply samples/sampler/stories/default/VideoPlayer.js]: It's fixed.

**Pull request merged**